CHANGELOG.md

```diff
@@ -3,7 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
-## [Unreleased]
+## [Unreleased
```

Comment:
Did we not like that final square bracket? :)

Reply:
ups, I think it was a result of an incomplete copying... :)